cli/internal/packager/seed.go

```diff
@@ -0,0 +1,161 @@
+package packager
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/defenseunicorns/zarf/cli/config"
+	"github.com/defenseunicorns/zarf/cli/internal/images"
+	"github.com/defenseunicorns/zarf/cli/internal/k8s"
+	"github.com/defenseunicorns/zarf/cli/internal/message"
+	"github.com/defenseunicorns/zarf/cli/internal/utils"
+	"github.com/distribution/distribution/v3/configuration"
+	"github.com/distribution/distribution/v3/registry"
+	_ "github.com/distribution/distribution/v3/registry/auth/htpasswd"             // used for embedde registry
+	_ "github.com/distribution/distribution/v3/registry/storage/driver/filesystem" // used for embedded registry
+
+	"github.com/sirupsen/logrus"
+)
+
+func startEmbeddedRegistry(readOnly bool, tls bool) {
+	registryConfig := &configuration.Configuration{}
+
+	if logrus.IsLevelEnabled(logrus.DebugLevel) {
```

Comment:
So this makes the log level either debug or error with no option to show the warnings from the registry? Is there a specific reason we're not giving an option to show warnings?

Reply:
Yeah distribution under the hood uses logrus too but it's logs for Info level are terrible and spew output everywhere.  Feel free to try it and tweak it, but I hated the log vomit with messed up line breaks from the registry.